mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule Protobuf.Mixfile do
     [
       app: :protobuf,
       version: @version,
-      elixir: "~> 1.6",
```

Comment:
If possible, let's not bump the version yet. I don't see a strong enough reason yet :)

Reply:
Good point. I've reverted the changes in #109 that required 1.8+ too.

Reply:
Done.